src/app.html

```diff
@@ -17,22 +17,22 @@
         %sveltekit.head%
     </head>
 
-    <body class="theme-dark" data-sveltekit-preload-data="hover">
```

Comment:
we might depend in some components on pinks light/dark theme selectors here, just a note that this could break something